package.json

```diff
@@ -10,7 +10,7 @@
   },
   "types": "./types.d.ts",
   "engines": {
-    "node": "^4 || ^5 || ^6 || ^7",
+    "node": "^4 || ^5 || ^6 || ^7 || ^8",
```

Comment:
If we do not test for 5 we should remove it from supported versions. However, since we still have known projects using 5, I would be cautious about this.